vispy/scene/widgets/axis.py

```diff
@@ -16,16 +16,32 @@ class AxisWidget(Widget):
     ----------
     orientation : str
         Orientation of the axis, 'left' or 'bottom'.
+    axis_mapping : array-like
+        Axis mapping maps a arbitrary array-like object of type numpy array of numpy.datetime64 or
+        list of datetime dates
+        All other types will be converted to strings
+        In case of string mapping the array/list will only be applied to integers
+        (0, 1, 2, 3, 4, 5, etc) not (0.5, 0.666, 0.75, 1.5, etc.) floats won't be displayed,
+        this is not the case if you provide a array of datetime64 or list of datetime dates
```

Comment:
```suggestion
        Axis mapping maps an arbitrary array-like object to the axis ticks. If a list of datetime objects
        or a numpy array of dtype numpy.datetime64, they will be interpolated; otherwise, elements will
        be cast to string and mapped to integer ticks.
```

Reply:
I would reword this as above, assuming that I udnerstood what it means.